ci/tests/e2e-helm.sh

```diff
@@ -29,6 +29,7 @@ oc version
 echo $ROOTDIR
 
 # install operator-sdk
+export GOBIN=${GOPATH}/bin
```

Comment:
Before `make install` there's an additional check if the `GOBIN` is set or not. For that reason this export is needed.
https://github.com/openshift/ocp-release-operator-sdk/pull/387/commits/011460111ef73896e64d17e4329c6b0407835ba1#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R76-R79

Reply:
A conditional check is required to set GOBIN only if there is no other value inherited from env.

Reply:
Added the check before setting the value of GOBIN.